src/components/StudentProfileBox/StudentProfileBox.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../common/vars.css';
+import { Button, Col } from 'react-bootstrap';
+import styles from './StudentProfileBox.module.css';
+
+const StudentProfileBox = ({ studentName }) => {
+  return (
+    <>
+      <Col className="d-flex justify-content-center align-items-center">{studentName}</Col>
+      <Col className="d-flex justify-content-center align-items-center">
+        <Button className={styles['view-profile-btn']}>View Profile</Button>
```

Comment:
I think you should also have a backend request here for the student info - you will need the `studentId` in order for the "view profile" button to work properly